Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var string[]
+     */
+    protected static $formats = array('json', 'xml', 'yml');
```

Comment:
> Or just

Both, I think.

> And since we are adding a deprecating warning why do we avoid from its message.

That's in case the class is used directly, without using the dependency injection at all (not sure this is possible though).